<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LogIn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RECUPERAR SENHA TESTE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetAltera">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButtom#loginButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 230, 219), stop:1 rgba(85, 81, 84, 226));
	color:rgba(255,255,255,210);
	border-radius: 5px; 

}

QPushButtom#loginButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButtom#loginButton:pressed{
	padding-left:5px;	
	padding-top:5px;
	background-color:rgba(150, 123, 111, 255);
}</string>
    </property>
    <property name="text">
     <string>ALTERAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="res_newpass">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
padding-bottom: 7px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Digite uma nova senha</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="res_newpass2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
padding-bottom: 7px;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Repita sua nova senha</string>
    </property>
   </widget>
   <widget class="QLabel" name="icon_pass">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Tela_login/password.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>res_newpass</tabstop>
  <tabstop>res_newpass2</tabstop>
  <tabstop>resetAltera</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
